Content/Tiles/Mushroom/JellyShroom.cs

```diff
@@ -18,8 +19,15 @@ class JellyShroom : DummyTile
 
         public override void SetStaticDefaults()
         {
-            // Is the sound correct..?
-            QuickBlock.QuickSetFurniture(this, 7, 7, DustType<Content.Dusts.BlueStamina>(), SoundID.PlayerHit, false, new Microsoft.Xna.Framework.Color(100, 200, 220), false, false, "Jelly Shroom");
+            QuickBlock.QuickSetFurniture(this, 7, 7, -1, SoundID.NPCDeath1, false, new Microsoft.Xna.Framework.Color(100, 200, 220), false, false, "Jelly Shroom");
```

Comment:
lets add a using directive for Microsoft.Xna.Framework instead of fully qualifying it here